snapshots/src/main/java/io/axoniq/dev/samples/command/MyEntityAggregate.java

```diff
@@ -0,0 +1,63 @@
+package io.axoniq.dev.samples.command;
+
+import io.axoniq.dev.samples.api.CreateMyEntityCommand;
+import io.axoniq.dev.samples.api.MyEntityCreatedEvent;
+import io.axoniq.dev.samples.api.MyEntityRenamedEvent;
+import io.axoniq.dev.samples.api.RenameMyEntityCommand;
+import org.axonframework.commandhandling.CommandHandler;
+import org.axonframework.eventsourcing.EventSourcingHandler;
+import org.axonframework.modelling.command.AggregateIdentifier;
+import org.axonframework.spring.stereotype.Aggregate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.invoke.MethodHandles;
+
+import static org.axonframework.modelling.command.AggregateLifecycle.apply;
+
+/**
+ * @author Sara Pellegrini
+ * @author Lucas Campos
+ */
+@Aggregate(snapshotTriggerDefinition = "mySnapshotTriggerDefinition")
+public class MyEntityAggregate {
+
+    private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+    @AggregateIdentifier
+    private String entityId;
+
+    private String name;
+
+    public MyEntityAggregate() {
+    }
+
+    @CommandHandler
+    public MyEntityAggregate(CreateMyEntityCommand command) {
+        apply(new MyEntityCreatedEvent(command.getEntityId(), command.getName()));
+        logger.info("[CreateMyEntityCommand] Entity with id [{}] and name [{}] created.",
+                    command.getEntityId(),
+                    command.getName());
+    }
+
+    @CommandHandler
+    public void on(RenameMyEntityCommand command) {
+        apply(new MyEntityRenamedEvent(command.getEntityId(), command.getName()));
```

Comment:
We can check here if the new name is different from previous one.

Reply:
Added.